voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.derpgroup.derpwizard.voice.model.alexa;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+
+@SuppressFBWarnings
+public class FlashBriefingResponse {
+  
+  public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'.0Z'");
```

Comment:
Didn't we want a format like this instead?
@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ")